infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -289,6 +289,25 @@ function ensure_staging_gcb() {
     ensure_project_role_binding "${project}" "${principal}" "roles/cloudbuild.builds.builder"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectCreator"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectViewer"
+
+    local sa_name="gcb-signer"
```

Comment:
```suggestion
    local sa_name="gcb-image-builder"
```

I feel like a more generic name make more sense since we do image building and signing.

Reply:
Good, fixed as suggested :+1: